rules/misc.lua

```diff
@@ -389,12 +389,12 @@ rspamd_config:register_symbol({
 rspamd_config:register_dependency('FREEMAIL_REPLYTO_NEQ_FROM_DOM', 'FREEMAIL_REPLYTO')
 rspamd_config:register_dependency('FREEMAIL_REPLYTO_NEQ_FROM_DOM', 'FREEMAIL_FROM')
 
-rspamd_config.OMOGRAPH_URL = {
+rspamd_config.HOMOGRAPH_URL = {
```

Comment:
This might break configs?

Reply:
Yes :( Please revert this one as it will break POLA likely.